.travis.yml

```diff
@@ -6,6 +6,8 @@ os:
 julia:
     - 1.2
     - 1.3
+    - 1.4
+    - 1.5
     - nightly
```

Comment:
Do you prefer to test all those versions? I usually do `1.2, 1, nightly` (1 is 1.5 now)

Reply:
Ok, I did not know that 1 resolves to 1.5. Yes, It probably makes sense not to test all of them. At one point we should have an appveyor test as well, in particular for 32bit systems.